### DISCUSSION TITLE
220902 일간회고

### DISCUSSION THREAD

**Discussion by yoonmi-cyber** [codestates/BEB-05-Team-Vote#34] [일간회고]
### 오늘은 어떻게 프로젝트에 기여했나요?

- TypeScript와 Next.js를 공부했습니다.

### 오늘의 프로젝트에서 힘든 점은 무엇인가요?

- TS에 비해서 Next.js는 좀 더 이해하기 쉬웠지만 빨리 학습하기 위해서 깊이 생각해보지 않았던 것은 아쉬웠습니다. 

### 내일은 프로젝트에 기여하기 위해 무엇을 해야 하나요?

- [ ] TS + Next.js로 페이지 구현을 위한 연습을 하고 styled-component, ant design을 사용해 보겠습니다.

**Comment by thyoondev** [comment]
너무 단기간에 많은 지식을 흡수하다보면 어느 순간 뇌가 거부하는 순간이 올 때도 있는 것 같아요 ㅠㅠ 
주말동안 푹 쉬고 다음 주 월요일부터 다시 달려보는건 어떨까요?

**Reply by yoonmi-cyber** [reply]
갓태희!! Jaden 감사합니다!👍🏼
